### PR TITLE
Documentation for mobile framework support

### PR DESCRIPTION
**Pull request by DarylThayil**
Add a doc to address questions we get around mobile support

### CODE REVIEW

**Comment by jasonnutter** [comment]
This might be better in the main README file, thoughts?

File: docs/markdown/js-mobile-frameworks.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+### Does Msal's Javascript platform support mobile frameworks such as cordova, ionic, or react native. 
+The short answer is, no.  The more nuanced answer is that while we do not support these frameworks explicitly, there is no reason that our libraries can not work with these frameworks.
+
+#### Making it work
+If the libraries do not work out of the box you may need to write some glue code to adapt our libraries to the framework.  If in your investigation you find some parameters / features that would make working with these frameworks easier, we accept contributions to our libraries.  We will not make any large archtiectural changes to support this, but if we can accomodate we will.
```
Comment (hectormmg) [inline docs/markdown/js-mobile-frameworks.md:5]:
```suggestion
If the libraries do not work out of the box you may need to write some glue code to adapt our libraries to the framework.  If in your investigation you find some parameters / features that would make working with these frameworks easier, we accept contributions to our libraries.  We will not make any large architectural changes to support this, but if we can accommodate we will.
```

**Comment by DarylThayil** [comment]
@jasonnutter  open to suggestions, maybe we can link from there but it wasn't clear where this belonged.

**Comment by jasonnutter** [comment]
I would be up for a repo-level FAQ (which could also then link to FAQs for each library).

**Comment by tnorling** [comment]
> I would be up for a repo-level FAQ (which could also then link to FAQs for each library).

+1 for this suggestion

**Pull request closed without merging**